Podfile

```diff
@@ -0,0 +1,12 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
```

Comment:
You'll notice, especially since you used Cocoapods, that you have many many files checked into the repo. In the future, using a nice .gitignore will remove much of the noise and unnecessary files to check into source code. A good example is of a gitignore for swift/xcode can be found here, notice there is a section where you can ignore certain cocoapod directories, which I would recommend doing. https://github.com/github/gitignore/blob/master/Swift.gitignore